nsqd/nsqd.go

```diff
@@ -502,12 +534,16 @@ exit:
 }
 
 func (n *NSQD) Notify(v interface{}) {
+	persist := !n.getFlag(flagLoading)
 	n.waitGroup.Wrap(func() {
 		// by selecting on exitChan we guarantee that
 		// we do not block exit, see issue #123
 		select {
 		case <-n.exitChan:
 		case n.notifyChan <- v:
+			if !persist {
```

Comment:
yes, since `Notify` may called by `NewTopic` /  `NewChannel`  in the loading stage